packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -449,6 +493,16 @@ class DescopeWc extends BaseDescopeWc {
       });
       return;
     }
+
+    this.logger().info(`Step "${stepName || `#${stepId}`}" is ${status}`, '', {
```

Comment:
this will make the log a bit noisi, by default, to every usage

maybe worth to split debugger and the dispatch? imo those are related, but not the same use cases

wdyt?

Reply:
i think this is currently very useful when running the flow. i would suggest we keep it and get used to it before we release. if we want before the release we can always remove the `console.log` or disable it

Reply:
how could you disable it?

I think it is useful if you are a Descoper that is debugging Descope flow, but if you are not, it is just noise, you may be a developer that is not interested in the login process, or something like this? 

I think that those two capabilities (a. debugging flow to subscribe to `onNext` and b. internal logging for troubleshooting purposes) are not tightly coupled

Reply:
i dont think we can easily assume it will cause noise and/or hard to work with.. this `info` logs that are normally filtered will only be displayed when actually running the flow and not developing your application. If we find this is too much noise we can address it in a way that fit the actual customer needs.

This pr only refers to `onLog`, the `onNext` is discarded and will be added at later stages to track the actual actions and screens

Reply:
than I somewhat I disagree about the need of `onLog`, I think that a developer would like to get whenever `next` happens and not when `log` happens, so I think those two topics should not be tightly coupled like they are now

>  this info logs that are normally filtered will only be displayed when actually running the flow and not developing your application

I don't think the average developer does this, but I argued enough, approving